PWGCF/MultiparticleCorrelations/Tasks/multiparticle-correlations-ab.cxx

```diff
@@ -74,6 +76,12 @@ bool gProcessSim_Run2 = false;
 bool gProcessRec_Run1 = false;
 bool gProcessRecSim_Run1 = false;
 bool gProcessSim_Run1 = false;
+bool gProcessTest = false; // minimum subscription to the tables, for testing purposes
```

Comment:
I forgot to mention this before but per each `processWhatEverName` process function within a `PROCESS_SWITCH` clause there is an implicit variable that is defined as `bool doprocessWhatEverName` that you can use in case you so consider

Reply:
Thanks, noted! I will integrate this feature into my code alongside the next PR.